src/cryptoadvance/specter/wallet.py

```diff
@@ -436,27 +438,45 @@ def _rescan_utxo_thread(self, explorer=None):
         ])
         # handle missing transactions now
         # if Tor is running, requests will be sent over Tor
+        tor_explorers = [
+            'http://explorerzydxu5ecjrkwceayqybizmpjjznk5izmitf2modhcusuqlid.onion',
+            'http://mempoolhqx4isw62xs7abwphsq7ldayuidyx2v2oethdhhj6mlo2r6ad.onion'
```

Comment:
These are mainnet only, on testnet requests will fail to fetch data.

Reply:
Fixed in f1ae57da1371a27e9368c1309ee72a0afe51e483